introduction.md

```diff
@@ -3,6 +3,6 @@ Hello world
 =====
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?